scripts/trlite

```diff
@@ -298,7 +298,7 @@ function check_jstring_util() {
     CALLCOUNT=$(grep jerry_string_to_char_buffer src/zjs_util.c | wc -l)
 
     # make sure there are exactly two calls to the function
-    if [ "$CALLCOUNT" != "2" ]; then
+    if [ $CALLCOUNT != "2" ]; then
```

Comment:
This is needed on Mac, because returned value after running wc -l contains some blank spaces before it, so if you double quote it, it became like "      2", and then comparison will fail